aiozipkin/span.py

```diff
@@ -177,3 +184,11 @@ def kind(self, span_kind: str) -> 'Span':
     def name(self, span_name: str) -> 'Span':
         self._record.name(span_name)
         return self
+
+    def new_child(self, name: str = None, kind: str = None) -> 'Span':
```

Comment:
what if we move this method to SpanAbc? As result we do not need implement it in base classes.

Reply:
I thought about it. But I did it for type hinting of real return classes